docs/_includes/Topics.md

```diff
@@ -1,5 +1,9 @@
 # Advanced Topics
 
+## Type Parameter Completion
+
+TO BE WRITTEN
```

Comment:
For now, you can add a link to the Dafny Power User note.

Reply:
Done